pkg/install/v1alpha1/cstor_volume_0.7.0.go

```diff
@@ -23,13 +23,7 @@ func CstorVolumeArtifactsFor070() (list ArtifactList) {
 	return
 }
 
-// cstorVolumeYamlsFor070 returns all the yamls related to cstor volume in a
-// string format
-//
-// NOTE:
-//  This is an implementation of MultiYamlFetcher
-func cstorVolumeYamlsFor070() string {
-	return `
+const cstorRunTask = `
```

Comment:
Name of constant is not appropriate.
Name of constant should make use of version no. as suffix.
The constant can be a typed string versus a string.